index.html

```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style.css">
-    <title>Trial Admission</title>
+    <title>Trial Admission Project</title>
 </head>
 
 <body>
```

Comment:
Good implementation of grid in this project container. 👍🏿 You almost achieved the design requirements.
I would however suggest that you consider applying the grid style to the mini-projects themselves.

 You can achieve that putting all the mini-projests in a container like <div> and giving the grid style to this container. 🎛️